django-daiquiri/daiquiri
# Refactoring, pytest and Django 4

* Remove archive, meetings and wordpress apps
* Use pytest for testing, refactor tests
* Upgrade to Django 4
* Fix fetch_table in PostgreSQLAdapter
* Fix error prone exception handling in BaseOaiAdapter
* Fix stream action in QueryJobViewSet
* Fix reject method in the Profile model
* Update Version (0.4.0), NOTICE, AUTHORS and README
* Pin antlr4-python3-runtime to 4.8
* Add .isort.cfg
